{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/gsingh/projects/VPESourceCode/env/lib/python3.10/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
      "  from .autonotebook import tqdm as notebook_tqdm\n"
     ]
    }
   ],
   "source": [
    "import basico"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<CDataModel \"Root\">\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'Teusink2000_Glycolysis'"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import os\n",
    "os.getcwd()\n",
    "model = basico.load_biomodel(64)\n",
    "# model = basico.load_model('/home/gsingh/projects/Gadkar2021.xml')\n",
    "print (model)\n",
    "# print (basico.biomodels.get_model_info(model)[\"description\"])\n",
    "# print (basico.model_info.get_notes(model=model))\n",
    "basico.model_info.get_model_name(model=model)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'name': 'Teusink2000_Glycolysis',\n",
       " 'description': '<notes xmlns=\"http://www.sbml.org/sbml/level2\">      <body xmlns=\"http://www.w3.org/1999/xhtml\">        <p>          <b>Can yeast glycolysis be understood in terms of in vitro kinetics of the constituent enzymes? Testing biochemistry.</b>          <br />          <i>Teusink,B et al.: Eur J Biochem 2000 Sep;267(17):5313-29.</i>          <br />          The model reproduces the steady-state fluxes and metabolite concentrations of the branched model as given in Table 4 of the paper. It is derived from the model on JWS online, but has the ATP consumption in the succinate branch with the same stoichiometrie as in the publication. The model was successfully tested on copasi v.4.4(build 26).      <br />          For Vmax values, please note that there is a conversion factor of approx. 270 to convert from U/mg-protein as shown in Table 1 of the paper to mmol/(min*L_cytosol). The equilibrium constant for the ADH reaction in the paper is given for the reverse reaction (Keq = 1.45*10      <sup>4</sup>          ). The value used in this model is for the forward reaction: 1/Keq = 6.9*10      <sup>-5</sup>          .      <br />          Vmax parameters values used (in [mM/min] except VmGLT):      <table border=\"0\" cellpadding=\"2\" cellspacing=\"0\">            <tr>              <td>                <b>VmGLT</b>              </td>              <td>97.264</td>              <td>mmol/min</td>            </tr>            <tr>              <td>                <b>VmGLK</b>              </td>              <td>226.45</td>              <td />            </tr>            <tr>              <td>                <b>VmPGI</b>              </td>              <td>339.667</td>              <td />            </tr>            <tr>              <td>                <b>VmPFK</b>              </td>              <td>182.903</td>              <td />            </tr>            <tr>              <td>                <b>VmALD</b>              </td>              <td>322.258</td>              <td />            </tr>            <tr>              <td>                <b>VmGAPDH_f</b>              </td>              <td>1184.52</td>              <td />            </tr>            <tr>              <td>                <b>VmGAPDH_r</b>              </td>              <td>6549.68</td>              <td />            </tr>            <tr>              <td>                <b>VmPGK</b>              </td>              <td>1306.45</td>              <td />            </tr>            <tr>              <td>                <b>VmPGM</b>              </td>              <td>2525.81</td>              <td />            </tr>            <tr>              <td>                <b>VmENO</b>              </td>              <td>365.806</td>              <td />            </tr>            <tr>              <td>                <b>VmPYK</b>              </td>              <td>1088.71</td>              <td />            </tr>            <tr>              <td>                <b>VmPDC</b>              </td>              <td>174.194</td>              <td />            </tr>            <tr>              <td>                <b>VmG3PDH</b>              </td>              <td>70.15</td>              <td />            </tr>          </table>          The result of the G6P steady state concentration (marked in red) differs slightly from the one given in table 4. of the publication      <br />          Results for steady state:      <table border=\"0\" cellpadding=\"2\" cellspacing=\"0\">            <thead>              <tr>                <th align=\"left\" bgcolor=\"#eeeeee\" valign=\"middle\" />                <th align=\"left\" bgcolor=\"#eeeeee\" valign=\"middle\">orig. article</th>                <th align=\"center\" bgcolor=\"#eeeeee\" valign=\"middle\">this model</th>              </tr>            </thead>            <tbody>              <tr>                <td>                  <b>Fluxes[mM/min]</b>                  \\xa0              </td>                <td />                <td />              </tr>              <tr>                <td>Glucose\\xa0</td>                <td>88\\xa0</td>                <td>88\\xa0</td>              </tr>              <tr>                <td>Ethanol\\xa0</td>                <td>129\\xa0</td>                <td>129\\xa0</td>              </tr>              <tr>                <td>Glycogen\\xa0</td>                <td>6\\xa0</td>                <td>6\\xa0</td>              </tr>              <tr>                <td>Trehalose\\xa0</td>                <td>4.8\\xa0</td>                <td>4.8\\xa0</td>                <td>(G6P flux through trehalose branch)</td>              </tr>              <tr>                <td>Glycerol\\xa0</td>                <td>18.2\\xa0</td>                <td>18.2\\xa0</td>              </tr>              <tr>                <td>Succinate\\xa0</td>                <td>3.6\\xa0</td>                <td>3.6\\xa0</td>              </tr>              <tr>                <td>                  <b>Conc.[mM]</b>                  \\xa0              </td>                <td />                <td />              </tr>              <tr>                <td>G6P\\xa0</td>                <td>1.07\\xa0</td>                <td style=\"color: red\">1.03\\xa0</td>              </tr>              <tr>                <td>F6P\\xa0</td>                <td>0.11\\xa0</td>                <td>0.11\\xa0</td>              </tr>              <tr>                <td>F1,6P\\xa0</td>                <td>0.6\\xa0</td>                <td>0.6\\xa0</td>              </tr>              <tr>                <td>DHAP\\xa0</td>                <td>0.74\\xa0</td>                <td>0.74\\xa0</td>              </tr>              <tr>                <td>3PGA\\xa0</td>                <td>0.36\\xa0</td>                <td>0.36\\xa0</td>              </tr>              <tr>                <td>2PGA\\xa0</td>                <td>0.04\\xa0</td>                <td>0.04\\xa0</td>              </tr>              <tr>                <td>PEP\\xa0</td>                <td>0.07\\xa0</td>                <td>0.07\\xa0</td>              </tr>              <tr>                <td>PYR\\xa0</td>                <td>8.52\\xa0</td>                <td>8.52\\xa0</td>              </tr>              <tr>                <td>AcAld\\xa0</td>                <td>0.17\\xa0</td>                <td>0.17\\xa0</td>              </tr>              <tr>                <td>ATP\\xa0</td>                <td>2.51\\xa0</td>                <td>2.51\\xa0</td>              </tr>              <tr>                <td>ADP\\xa0</td>                <td>1.29\\xa0</td>                <td>1.29\\xa0</td>              </tr>              <tr>                <td>AMP\\xa0</td>                <td>0.3\\xa0</td>                <td>0.3\\xa0</td>              </tr>              <tr>                <td>NAD\\xa0</td>                <td>1.55\\xa0</td>                <td>1.55\\xa0</td>              </tr>              <tr>                <td>NADH\\xa0</td>                <td>0.04\\xa0</td>                <td>0.04\\xa0</td>              </tr>            </tbody>          </table>          Authors of the publication also mentioned a few misprints in the original article:      <br />          in the kinetic law for      <em>ADH</em>          :      <ol>            <li>the species          <em>a</em>              should denote          <em>NAD</em>              and          <em>b</em>            <em>Ethanol</em>        </li>        <li>the last term in the equation should read          <em>bpq</em>              /(          <em>K            <sub>ib</sub>                K            <sub>iq</sub>                K            <sub>p</sub>    </em>              )          </li></ol>          in the kinetic law for      <em>PFK</em>          :      <ol>  <li>R = 1 + λ          <sub>1</sub>              + λ          <sub>2</sub>              + g          <sub>r</sub>              λ          <sub>1</sub>              λ          <sub>2</sub></li><li>equation L  should read: L = L0*(..)          <sup>2</sup>              *(..)          <sup>2</sup>              *(..)          <sup>2</sup>              not L = L0*(..)          <sup>2</sup>              *(..)          <sup>2</sup>              *(..)          </li></ol>          To make the model easier to curate, the species      <em>ATP</em>          ,      <em>ADP</em>          and      <em>AMP</em>          were added. These are calculated via assignment rules from the active phosphate species,      <em>P</em>          , and the sum of all      <em>AXP</em>          ,      <em>SUM_P</em>          .      </p><br /><p>To the extent possible under law, all copyright and related or neighbouring rights to this encoded model have been dedicated to the public domain worldwide. Please refer to      <a href=\"http://creativecommons.org/publicdomain/zero/1.0/\" title=\"Creative Commons CC0\">CC0 Public Domain Dedication</a>          for more information.      </p><p>In summary, you are entitled to use this encoded model in absolutely any manner you deem suitable, verbatim, or with modification, alone or embedded it in a larger context, redistribute it, commercially or not, in a restricted way or not.</p><br /><p>To cite BioModels Database, please use:      <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/20587024\" target=\"_blank\">Li C, Donizelli M, Rodriguez N, Dharuri H, Endler L, Chelliah V, Li L, He E, Henry A, Stefan MI, Snoep JL, Hucka M, Le Novère N, Laibe C (2010) BioModels Database: An enhanced, curated and annotated resource for published quantitative kinetic models. BMC Syst Biol., 4:92.</a></p></body></notes>',\n",
       " 'format': {'name': 'SBML', 'identifier': 'SBML', 'version': 'L2V1'},\n",
       " 'publication': {'type': 'PubMed ID',\n",
       "  'accession': '10951190',\n",
       "  'journal': 'European journal of biochemistry',\n",
       "  'title': 'Can yeast glycolysis be understood in terms of in vitro kinetics of the constituent enzymes? Testing biochemistry.',\n",
       "  'affiliation': 'E.C. Slater Institute, BioCentrum Amsterdam, University of Amsterdam, the Netherlands.',\n",
       "  'synopsis': 'This paper examines whether the in vivo behavior of yeast glycolysis can be understood in terms of the in vitro kinetic properties of the constituent enzymes. In nongrowing, anaerobic, compressed Saccharomyces cerevisiae the values of the kinetic parameters of most glycolytic enzymes were determined. For the other enzymes appropriate literature values were collected. By inserting these values into a kinetic model for glycolysis, fluxes and metabolites were calculated. Under the same conditions fluxes and metabolite levels were measured. In our first model, branch reactions were ignored. This model failed to reach the stable steady state that was observed in the experimental flux measurements. Introduction of branches towards trehalose, glycogen, glycerol and succinate did allow such a steady state. The predictions of this branched model were compared with the empirical behavior. Half of the enzymes matched their predicted flux in vivo within a factor of 2. For the other enzymes it was calculated what deviation between in vivo and in vitro kinetic characteristics could explain the discrepancy between in vitro rate and in vivo flux.',\n",
       "  'year': 2000,\n",
       "  'month': '9',\n",
       "  'volume': '267',\n",
       "  'issue': '17',\n",
       "  'pages': '5313-5329',\n",
       "  'link': 'http://identifiers.org/pubmed/10951190',\n",
       "  'authors': [{'name': 'B Teusink',\n",
       "    'institution': 'Systems Bioinformatics, Amsterdam Institute for Molecules, Medicines and Systems, VU Amsterdam, Amsterdam, The Netherlands.',\n",
       "    'orcid': '0000-0003-3929-0423'},\n",
       "   {'name': 'J Passarge'},\n",
       "   {'name': 'C A Reijenga'},\n",
       "   {'name': 'E Esgalhado'},\n",
       "   {'name': 'C C van der Weijden', 'orcid': '0000-0001-8663-7579'},\n",
       "   {'name': 'M Schepper'},\n",
       "   {'name': 'M C Walsh'},\n",
       "   {'name': 'B M Bakker',\n",
       "    'institution': 'Microbial Physiology, BioCentrum Amsterdam, Vrije Universiteit, De Boelelaan 1087, NL-1081 HV Amsterdam, BioCentrum Amsterdam, University of Amsterdam, Plantage Muidergracht 12, NL-1018 TV Amsterdam, The Netherlands.',\n",
       "    'orcid': '0000-0001-6274-3633'},\n",
       "   {'name': 'K van Dam'},\n",
       "   {'name': 'H V Westerhoff',\n",
       "    'institution': 'Infrastructure for Systems Biology Europe - The Netherlands (ISBE.NL), Amsterdam, The Netherlands. H.V.Westerhoff@VU.NL.Molecular Cell Biology, VU University Amsterdam, Amsterdam, The Netherlands. H.V.Westerhoff@VU.NL.Synthetic Systems Biology and Nuclear Organization, Swammerdam Institute for Life Sciences, University of Amsterdam, Amsterdam, The Netherlands. H.V.Westerhoff@VU.NL.Manchester Centre for Integrative Systems Biology, Manchester, UK. H.V.Westerhoff@VU.NL.',\n",
       "    'orcid': '0000-0002-0443-6114'},\n",
       "   {'name': 'Snoep JL', 'orcid': '0000-0002-0405-8854'}]},\n",
       " 'files': {'main': [{'name': 'BIOMD0000000064_url.xml',\n",
       "    'description': 'SBML L2V1 representation of Teusink2000_Glycolysis',\n",
       "    'fileSize': '103423',\n",
       "    'mimeType': 'application/xml',\n",
       "    'md5sum': 'a4fe8e3d30d672cebdc131c187ad9472',\n",
       "    'sha1sum': '82ebc192deb948150a0a802d36e1b6cef579ee0a',\n",
       "    'sha256sum': 'd6a75df939d6707fdb904b44c1f530c8dc3600fd44267ff8e8f38dac560b4361'}],\n",
       "  'additional': [{'name': 'BIOMD0000000064-biopax2.owl',\n",
       "    'description': 'Auto-generated BioPAX (Level 2)',\n",
       "    'fileSize': '60631',\n",
       "    'mimeType': 'application/rdf+xml',\n",
       "    'md5sum': '1b6a9c866cf8bf987f93ac93980121d4',\n",
       "    'sha1sum': 'af4237cab6e6ade3cb120692b9c4692f520e5e59',\n",
       "    'sha256sum': '8f400f7f5e82b4323f6d433b9f602834a0dc4a1d2248675e3181353295ad8820'},\n",
       "   {'name': 'BIOMD0000000064-biopax3.owl',\n",
       "    'description': 'Auto-generated BioPAX (Level 3)',\n",
       "    'fileSize': '82943',\n",
       "    'mimeType': 'application/rdf+xml',\n",
       "    'md5sum': '45ae2b34f03b14b9d46d5653f4182751',\n",
       "    'sha1sum': '0935e88c508b94843d59ff10bc2479ec0b4ac5d4',\n",
       "    'sha256sum': '19281d11119d9b98366649a23c08983463c9881fc1272ac44a7767c692a4d88b'},\n",
       "   {'name': 'BIOMD0000000064-matlab.m',\n",
       "    'description': 'Auto-generated Matlab file.',\n",
       "    'fileSize': '20846',\n",
       "    'mimeType': 'text/x-matlab',\n",
       "    'md5sum': '844f3b5ec868a34c92cf0ace00e40038',\n",
       "    'sha1sum': 'd88ed7a78c32ea7346969411153376ebbf65bd73',\n",
       "    'sha256sum': '4e0338145384a2f5b17c0df9b1d73d7fea192119ccb364e0069b400507208991'},\n",
       "   {'name': 'BIOMD0000000064-octave.m',\n",
       "    'description': 'Auto-generated Octave file.',\n",
       "    'fileSize': '20846',\n",
       "    'mimeType': 'text/x-matlab',\n",
       "    'md5sum': '962eff49b3e21ad8b7cab27e07a9692e',\n",
       "    'sha1sum': '2b813235ca440b512343e8312726097eda2ac5e3',\n",
       "    'sha256sum': 'a3e1e5a7c74468ce8c8fa2486fc1b40e03a88ed7db4757369288dd0b7067c2c1'},\n",
       "   {'name': 'BIOMD0000000064.m',\n",
       "    'description': 'Auto-generated Octave file',\n",
       "    'fileSize': '20846',\n",
       "    'mimeType': 'text/x-matlab',\n",
       "    'md5sum': '962eff49b3e21ad8b7cab27e07a9692e',\n",
       "    'sha1sum': '2b813235ca440b512343e8312726097eda2ac5e3',\n",
       "    'sha256sum': 'a3e1e5a7c74468ce8c8fa2486fc1b40e03a88ed7db4757369288dd0b7067c2c1'},\n",
       "   {'name': 'BIOMD0000000064.ode',\n",
       "    'description': 'Auto-generated ODE file for XPP.',\n",
       "    'fileSize': '15156',\n",
       "    'mimeType': 'text/plain',\n",
       "    'md5sum': '03704b5671703f539106bf5d9a0a825a',\n",
       "    'sha1sum': 'dbc79ebbe42679a22f9a6cf6ec120b70b83028d5',\n",
       "    'sha256sum': '4ee98f9c05c11f8e27d7108b259ed9097681fc0e62ffd876c15da16d36d2627f'},\n",
       "   {'name': 'BIOMD0000000064.pdf',\n",
       "    'description': 'Auto-generated PDF file',\n",
       "    'fileSize': '708922',\n",
       "    'mimeType': 'application/pdf',\n",
       "    'md5sum': '547940828c2cc1c79acd06c9a6489961',\n",
       "    'sha1sum': '6883ab0903104c4ad34f4d8886eb0660dc009fd2',\n",
       "    'sha256sum': 'a644146c3e0600dac3a3cba43f85121629ea75e835fc3103bc67e0c00bcd14db'},\n",
       "   {'name': 'BIOMD0000000064.png',\n",
       "    'description': 'Auto-generated Reaction graph (PNG)',\n",
       "    'fileSize': '228084',\n",
       "    'mimeType': 'image/png',\n",
       "    'md5sum': 'b114a152c46744538fbe35d7bd7bd06e',\n",
       "    'sha1sum': '2ace889b7e90e4a48d7fe00a779f98add0d32c57',\n",
       "    'sha256sum': '841d5f933e44fb21d42b3c666d09871ec99a5fd5fe6e0572cdf110e50f625710'},\n",
       "   {'name': 'BIOMD0000000064.svg',\n",
       "    'description': 'Auto-generated Reaction graph (SVG)',\n",
       "    'fileSize': '46271',\n",
       "    'mimeType': 'image/svg+xml',\n",
       "    'md5sum': 'f44e9827b1a308879c460f8c3edcfb64',\n",
       "    'sha1sum': 'da98ef84fc5a7540eb9b9f680d273c7df5746321',\n",
       "    'sha256sum': 'acff564e9e55b45e5c9f66eef43a924334af2e917da2f180de41f9c0f165d5d7'},\n",
       "   {'name': 'BIOMD0000000064.vcml',\n",
       "    'description': 'Auto-generated VCML file',\n",
       "    'fileSize': '121433',\n",
       "    'mimeType': 'application/xml',\n",
       "    'md5sum': '4889bb05db24f4ca8065a0b04f9f92b1',\n",
       "    'sha1sum': 'ffed2a36ad5f0aac4534220fb95deb159a904b57',\n",
       "    'sha256sum': '072c60b8e802630d2f7a46effcc72a2bc7e9f7d5b700ae75f93d430fa984e100'},\n",
       "   {'name': 'BIOMD0000000064_url.sedml',\n",
       "    'description': 'CRBM auto-generated \"template\" SED-ML file.',\n",
       "    'fileSize': '36134',\n",
       "    'mimeType': 'application/xml',\n",
       "    'md5sum': '706e6c6087cb5ebd3f7b92e851064e24',\n",
       "    'sha1sum': '6a39d4bb9e29e384d1a23a78e7d1fc0b7ca3d857',\n",
       "    'sha256sum': '65efd794a02e181d2da4eb3b05ada312dad63f8c511c61b4df5bf5536fb15d6a'},\n",
       "   {'name': 'curation_image.png',\n",
       "    'description': 'Reproduced figure(s) from original curation.',\n",
       "    'fileSize': '7882',\n",
       "    'mimeType': 'image/png',\n",
       "    'md5sum': '3d49892b06b3d037df8818710956d56a',\n",
       "    'sha1sum': '72dee3476623e277fe2f6c5f644f22302df56285',\n",
       "    'sha256sum': '81b2c5cd1661728689c957fe080fdaedd58d1cfa2279ca0437a2654c961ed861'},\n",
       "   {'name': 'curation_notes.txt',\n",
       "    'description': 'Notes from original curation (describes curation_image).',\n",
       "    'fileSize': '169',\n",
       "    'mimeType': 'text/plain',\n",
       "    'md5sum': 'b2f9804544a3f6719343dd7245f910de',\n",
       "    'sha1sum': '2b4e84c3a4881423d7511683fc796a913d7fade1',\n",
       "    'sha256sum': 'b0dfad3ce5e2905ae39f7a956a1e7c222610573cd5fd5467c5fdd156b3010dd0'},\n",
       "   {'name': 'manifest.xml',\n",
       "    'description': 'Auto-generated manifest file for OMEX archives.',\n",
       "    'fileSize': '1912',\n",
       "    'mimeType': 'application/xml',\n",
       "    'md5sum': '4dbaf7a102b603b039d738371e674d1b',\n",
       "    'sha1sum': '6d2b3805605bdf1e7bf0857109de2d3d424baef2',\n",
       "    'sha256sum': 'ef9ca8e2d4f3169edfafa9fb4e9e0b647cf6f8283f90774718ef9c4df3c29674'},\n",
       "   {'name': 'metadata.rdf',\n",
       "    'description': 'Auto-generated annotation metadata file.',\n",
       "    'fileSize': '13322',\n",
       "    'mimeType': 'application/rdf+xml',\n",
       "    'md5sum': 'b5628568165470f4d537efd3d4c06895',\n",
       "    'sha1sum': 'fc9add45ed7e41d61be97710fb517b21d381ac32',\n",
       "    'sha256sum': 'cbf4a4efa4e72b3ca118feadde9e79690ad37e9b064fe7e95c8a76161219b1b5'}]},\n",
       " 'history': {'revisions': [{'version': 1,\n",
       "    'submitted': 1155546330,\n",
       "    'submitter': 'Nicolas Le Novère',\n",
       "    'comment': 'Original import of Teusink2000_Glycolysis'},\n",
       "   {'version': 2,\n",
       "    'submitted': 1342722367,\n",
       "    'submitter': 'Nicolas Le Novère',\n",
       "    'comment': 'Current version of Teusink2000_Glycolysis'},\n",
       "   {'version': 3,\n",
       "    'submitted': 1724262808,\n",
       "    'submitter': 'Lucian Smith',\n",
       "    'comment': 'CRBM-sponsored manual and automated updates.'}]},\n",
       " 'firstPublished': 1725281262,\n",
       " 'submissionId': 'MODEL6623915522',\n",
       " 'publicationId': 'BIOMD0000000064',\n",
       " 'modellingApproach': {'accession': 'MAMO_0000046',\n",
       "  'name': 'ordinary differential equation model',\n",
       "  'resource': 'http://identifiers.org/mamo/MAMO_0000046'},\n",
       " 'curationStatus': 'CURATED',\n",
       " 'contributors': {'Curator': [{'name': 'Lucian Smith',\n",
       "    'email': 'lpsmith@uw.edu',\n",
       "    'affiliation': 'University of Washington',\n",
       "    'orcid': '0000-0001-7002-6386',\n",
       "    'external': False}],\n",
       "  'Modeller': [{'name': 'Nicolas Le Novère',\n",
       "    'email': 'lenov@ebi.ac.uk',\n",
       "    'affiliation': 'EBML-EBI',\n",
       "    'orcid': '0000-0002-6309-7327',\n",
       "    'external': False}]},\n",
       " 'vcsIdentifier': 'aaa',\n",
       " 'modelLevelAnnotations': [{'qualifier': 'bqbiol:hasTaxon',\n",
       "   'accession': '4932',\n",
       "   'name': 'Saccharomyces cerevisiae',\n",
       "   'resource': 'Taxonomy',\n",
       "   'uri': 'http://identifiers.org/taxonomy/4932'},\n",
       "  {'qualifier': 'bqbiol:is',\n",
       "   'accession': 'sce00010',\n",
       "   'name': 'Glycolysis / Gluconeogenesis - Saccharomyces cerevisiae (budding yeast)',\n",
       "   'resource': 'KEGG Pathway',\n",
       "   'uri': 'http://identifiers.org/kegg.pathway/sce00010'},\n",
       "  {'qualifier': 'bqmodel:is',\n",
       "   'accession': 'MODEL6623915522',\n",
       "   'resource': 'BioModels Database',\n",
       "   'uri': 'http://identifiers.org/biomodels.db/MODEL6623915522'},\n",
       "  {'qualifier': 'bqmodel:is',\n",
       "   'accession': 'BIOMD0000000064',\n",
       "   'resource': 'BioModels Database',\n",
       "   'uri': 'http://identifiers.org/biomodels.db/BIOMD0000000064'},\n",
       "  {'qualifier': 'bqmodel:isDescribedBy',\n",
       "   'accession': '10951190',\n",
       "   'resource': 'PubMed',\n",
       "   'uri': 'http://identifiers.org/pubmed/10951190'},\n",
       "  {'qualifier': 'bqbiol:is',\n",
       "   'accession': 'GO:0006096',\n",
       "   'name': 'glycolytic process',\n",
       "   'resource': 'Gene Ontology',\n",
       "   'uri': 'http://identifiers.org/go/GO:0006096'},\n",
       "  {'qualifier': 'bqbiol:isHomologTo',\n",
       "   'accession': 'REACT_723',\n",
       "   'name': 'Glucose metabolism',\n",
       "   'resource': 'Reactome',\n",
       "   'uri': 'http://identifiers.org/reactome/REACT_723'},\n",
       "  {'qualifier': 'bqbiol:hasProperty',\n",
       "   'accession': 'MAMO_0000046',\n",
       "   'name': 'Ordinary differential equation model',\n",
       "   'resource': 'Mathematical Modelling Ontology',\n",
       "   'uri': 'http://identifiers.org/mamo/MAMO_0000046'}]}"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from basico import *\n",
    "basico.biomodels.get_model_info(64)\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "env",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
